tests/test_all_dichotomies.py

```diff
@@ -0,0 +1,26 @@
+import pytest
+
+from tests.conftest import trained_decoder
+
+
+@pytest.mark.filterwarnings("ignore::sklearn.exceptions.ConvergenceWarning")
+@pytest.mark.parametrize("trained_decoder", [("base", "base"),
+                                              ("base", "svc"),
+                                             ("base", "nonlinear")
+                                              ], indirect=["trained_decoder"])
+def test_variable_dichotomies(trained_decoder) -> None:
+    pvals = [result.pval for key, result in iter(trained_decoder.results) if key != "XOR"]
+    assert all(pval < 0.05 for pval in pvals), \
+        (f"Variable dichotomies should able to be decoded in a low-dimensional geometry,"
+         f"{trained_decoder.key=}")
+
+
+@pytest.mark.filterwarnings("ignore::sklearn.exceptions.ConvergenceWarning")
+@pytest.mark.parametrize("trained_decoder", [("base", "base"),
+                                              ("base", "svc")
+                                              ], indirect=["trained_decoder"])
+def test_xor_dichotomies(trained_decoder) -> None:
```

Comment:
this should be singular - XOR is just one dichotomy in the case of 2 variables (which is the case)

Reply:
"XOR" has now been included in `TestDecoderCase.test_non_linearly_separable_semantic_dichotomies`. I've labeled it as plural because it could be the case that a specific test case could have multiple dichotomies that are only decodable using a non-linear classifier.